docs/docs/guides/labs/components/components-etl-pipeline-tutorial.md

```diff
@@ -49,14 +49,30 @@ See the [`duckdb`](https://duckdb.org/docs/installation/?version=stable&environm
 
 ### 2. Scaffold a new project
 
-After installing dependencies, scaffold a components-ready project:
+After installing dependencies, scaffold a components-ready project. The flow for scaffolding a project will depend on your package manager/environment management strategy.
 
-<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-scaffold.txt" />
-
-The `dg scaffold project` command builds a project at `jaffle-platform` and initializes a new Python
-virtual environment inside it. When you use `dg`'s default environment management behavior, you won't need to worry about activating this virtual environment yourself.
+<Tabs>
+    <TabItem value="uv" label="`uv` (global `dg`)">
```

Comment:
Labels should just be "uv" and "pip". We can explain the global install in the text of the uv section